python/tskit/util.py

```diff
@@ -316,6 +316,38 @@ def obj_to_collapsed_html(d, name=None, open_depth=0):
         return f"{name} {d}"
 
 
+def unicode_table(title, header, rows):
+    if header:
+        all_rows = [header] + rows
+    else:
+        all_rows = rows
+    widths = [
+        max(len(row[i_col]) for row in all_rows) for i_col in range(len(all_rows[0]))
+    ]
+    out = []
+    if title is not None:
+        w = sum(widths) + len(rows[1]) - 1
+        out += [
+            f"╔{'═' * w}╗\r\n" f"║{title.ljust(w)}║\r\n",
```

Comment:
Why the ``\r``s here? I think Python handles the universal newlines stuff and we can just put in ``\n``.

Reply:
Cool! Removed them all.